knowledge/astronomy/constellations/Phoenix/qna.yaml

```diff
@@ -0,0 +1,23 @@
+created_by: 
+domain: constellation
+seed_examples:
+- answer: |
```

Comment:
Thank you for your submission; taking a look at your `qna.yaml`, we noticed that you have some issues with the multiline `yaml` syntax.

Can you take a look at: https://yaml-multiline.info/ and fix up the file? Thanks!